tcpserver/tcpserver.go

```diff
@@ -1,75 +1,74 @@
 package tcpserver
 
 import (
-    "strconv"
-    "net"
-    "log"
-    "time"
-    "bufio"
-	"io"
+	"bufio"
 	"fmt"
+	"io"
+	"log"
+	"net"
+	"strconv"
+	"time"
 )
 
-type Handler struct {
-    conn   net.Conn
-    closed chan bool
+type Handler struct { // Handler : describe what this function does
```

Comment:
I think this comment does not apply

Reply:
👍 , I will add a proper description (I forgot it)